torchgeo/datamodules/nasa_marine_debris.py

```diff
@@ -45,12 +44,10 @@ def __init__(
         self.val_split_pct = val_split_pct
         self.test_split_pct = test_split_pct
 
-        self.aug = AugPipe(
-            AugmentationSequential(
-                K.Normalize(mean=self.mean, std=self.std), data_keys=["image", "boxes"]
-            ),
-            batch_size,
+        self.aug = K.AugmentationSequential(
+            K.Normalize(mean=self.mean, std=self.std), data_keys=None, keepdim=True
         )
+        self.aug.keepdim = True  # type: ignore[attr-defined]
```

Comment:
For now, the attribute has to be set manually. Refer to kornia/kornia#2848

Reply:
Let's add a comment here like we did in the other PR, I won't remember this even a week from now.

Reply:
Same thing every other location we need this. Hopefully the upstream bug gets resolved soon and we can remove all these comments.